packages/gatsby-plugin-utils/src/node-api-is-supported.ts

```diff
@@ -0,0 +1,12 @@
+export function isGatsbyNodeLifecycleSupported(apiName: string): boolean {
+  let availableAPIs
+  try {
+    availableAPIs = require(`gatsby/apis.json`)
+  } catch (e) {
+    throw new Error(
+      `Couldn't check available APIs. Make sure you are on gatsby version >=2.13.41`
+    )
+  }
+
+  return !!availableAPIs?.node?.[apiName]
+}
```

Comment:
speculative helper function for plugins to do runtime checks what's available with current version of gatsby core installed